lccs_db/__init__.py

```diff
@@ -7,8 +7,14 @@
 #
 
 """Python Land Cover Classification System Database Model."""
+import mimetypes
 
 from .ext import LCCSDatabase
 from .version import __version__
 
-__all__ = ('__version__', 'LCCSDatabase',)
\ No newline at end of file
+mimetypes.init()
+mimetypes.add_type("application/qml ", ".qml", True)
```

Comment:
Check if the is a standard for QGIS QML files, otherwise use `application/xml`.

Reply:
We didn't find a standard for the qml format